<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageB</class>
 <widget class="QWidget" name="PageB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>731</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic/all.qrc">:/pic/PageB-bg.png</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1131</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane{
min-width:100px;
min-height:50px;
border-top: 2px solid;

}

QTabBar::tab {

min-width:100px;

min-height:50px;

color: white;

font:20px &quot;Microsoft YaHei&quot;;

border: 0px solid;



}

QTabBar::tab:selected{

min-width:100px;

min-height:50px;
color: white;

font:22px &quot;Microsoft YaHei&quot;;

border: 0px solid;

border-bottom: 2px solid;

border-color: #4796f0;

}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>游览</string>
    </attribute>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView {
	background-color: transparent;
	font: 7pt &quot;Microsoft YaHei&quot;;
	/*font-weight:bold;*/
	border: none;
}

QListView::item {
    height: 30px;
}

QListView::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}

QListView::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="t1pic02">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>200</y>
       <width>275</width>
       <height>422</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB-02.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="t1pic03">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>200</y>
       <width>275</width>
       <height>422</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB-03.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="t1pic01">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>275</width>
       <height>421</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB-01.png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="t1le">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: transparent;
	border: none;
	text-align: center; 
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="t1btn_search">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-search-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-search-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="t1cb_searchType">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>85</y>
       <width>151</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    border: none;
    border-radius:22px;   /* 圆角 */
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */
    /*color: rgba(0,0,0,1);*/
    font: normal normal 20px &quot;Microsoft YaHei&quot;;
    background: transparent;
    text-align: AlignHCenter;
 
    color:rgb(255,255,255);/*字体颜色*/
    background-color: transparent;
    border-image: url(:/pic/PageB2-combo-bg1.png);/*背景图片，圆角*/
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border-radius:10px;   /* 圆角 */
    padding-top:10px;    /*上下都留10px，是因为选中项是矩形，如果不留，移动到第一项和最后一项会变为直角，不好看*/
    padding-bottom:10px;
 
    color:rgb(160,160,160);
    border-image: url(:/pic/PageB2-combo-bg2.png);/*下拉框背景，圆角图片*/
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
 
    min-height: 47px;/*每项高度*/
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255);
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
 
    border-image: url(:/pic/PageB2-combo-bg1.png);/*鼠标悬浮时背景图片*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}

/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/pic/down.png);/*自定义图片填充*/
	width: 15px;/*设置该图标的宽高*/
	height: 15px;
}

</string>
     </property>
     <item>
      <property name="text">
       <string>名称</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>类别</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>关键字</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="t1pic_search">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>44</y>
       <width>651</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB-searchLine.png</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="t1cb_sortType">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    /*border: 1px solid gray;   /* 边框 */
    border-radius:15px;   /* 圆角 */
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */
    color: rgba(0,0,0,1);
    font: normal normal 20px &quot;Microsoft YaHei&quot;;
    background: transparent;
    text-align: AlignHCenter;
 
    color:rgb(255,255,255);/*字体颜色*/
    background-color: transparent;
   /* border-image: url(:/pic/PageB2-combo-bg2.png);/*背景图片，圆角*/
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border-radius:10px;   /* 圆角 */
    padding-top:10px;    /*上下都留10px，是因为选中项是矩形，如果不留，移动到第一项和最后一项会变为直角，不好看*/
    padding-bottom:10px;
 
    color:rgb(160,160,160);
    border-image: url(:/pic/PageB2-combo-bg2.png);/*下拉框背景，圆角图片*/
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
 
    min-height: 50px;/*每项高度*/
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255);
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
 
    border-image: url(:/pic/PageB2-combo-bg1.png);/*鼠标悬浮时背景图片*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}

/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/pic/down.png);/*自定义图片填充*/
	width: 15px;/*设置该图标的宽高*/
	height: 15px;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>按热度排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按评价排序</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>310</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView {
	background-color: transparent;
	font: 7pt &quot;Microsoft YaHei&quot;;
	border: none;
}

QListView::item {
    height: 30px;
}

QListView::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}

QListView::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>310</y>
       <width>231</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView {
	background-color: transparent;
	font: 7pt &quot;Microsoft YaHei&quot;;
	border: none;
}

QListView::item {
    height: 30px;
}

QListView::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid rgb(130, 130, 130);
}

QListView::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="t1scrollArea">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>1041</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#scrollAreaWidgetContents{
    background:transparent;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="t1scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1039</width>
        <height>429</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents{
    background:transparent;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="t1le_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>610</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: transparent;
	border: none;
	text-align: center; 
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="t1picnoans">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>977</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB1-noans.png</pixmap>
     </property>
    </widget>
    <zorder>t1pic_search</zorder>
    <zorder>t1le</zorder>
    <zorder>t1pic02</zorder>
    <zorder>t1pic03</zorder>
    <zorder>t1pic01</zorder>
    <zorder>listWidget</zorder>
    <zorder>t1cb_searchType</zorder>
    <zorder>t1cb_sortType</zorder>
    <zorder>t1btn_search</zorder>
    <zorder>listWidget_2</zorder>
    <zorder>listWidget_3</zorder>
    <zorder>t1scrollArea</zorder>
    <zorder>t1le_2</zorder>
    <zorder>t1picnoans</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>笔记</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>1111</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#scrollAreaWidgetContents{
    background:transparent;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1109</width>
        <height>449</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#scrollAreaWidgetContents{
    background:transparent;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="t2pic_search">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>44</y>
       <width>651</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB-searchLine.png</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="t2cb_searchType">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>85</y>
       <width>151</width>
       <height>47</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    border: none;
    border-radius:22px;   /* 圆角 */
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */
    /*color: rgba(0,0,0,1);*/
    font: normal normal 20px &quot;Microsoft YaHei&quot;;
    background: transparent;
    text-align: AlignHCenter;
 
    color:rgb(255,255,255);/*字体颜色*/
    background-color: transparent;
    border-image: url(:/pic/PageB2-combo-bg1.png);/*背景图片，圆角*/
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border-radius:10px;   /* 圆角 */
    padding-top:10px;    /*上下都留10px，是因为选中项是矩形，如果不留，移动到第一项和最后一项会变为直角，不好看*/
    padding-bottom:10px;
 
    color:rgb(160,160,160);
    border-image: url(:/pic/PageB2-combo-bg2.png);/*下拉框背景，圆角图片*/
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
 
    min-height: 47px;/*每项高度*/
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255);
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
 
    border-image: url(:/pic/PageB2-combo-bg1.png);/*鼠标悬浮时背景图片*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}

/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/pic/down.png);/*自定义图片填充*/
	width: 15px;/*设置该图标的宽高*/
	height: 15px;
}

</string>
     </property>
     <item>
      <property name="text">
       <string>日记标题</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>日记内容</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>目的地</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="t2cb_sortType">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    /*border: 1px solid gray;   /* 边框 */
    border-radius:15px;   /* 圆角 */
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */
    color: rgba(0,0,0,1);
    font: normal normal 20px &quot;Microsoft YaHei&quot;;
    background: transparent;
    text-align: AlignHCenter;
 
    color:rgb(255,255,255);/*字体颜色*/
    background-color: transparent;
   /* border-image: url(:/pic/PageB2-combo-bg2.png);/*背景图片，圆角*/
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border-radius:10px;   /* 圆角 */
    padding-top:10px;    /*上下都留10px，是因为选中项是矩形，如果不留，移动到第一项和最后一项会变为直角，不好看*/
    padding-bottom:10px;
 
    color:rgb(160,160,160);
    border-image: url(:/pic/PageB2-combo-bg2.png);/*下拉框背景，圆角图片*/
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
 
    min-height: 50px;/*每项高度*/
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255);
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
 
    border-image: url(:/pic/PageB2-combo-bg1.png);/*鼠标悬浮时背景图片*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}

/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/pic/down.png);/*自定义图片填充*/
	width: 15px;/*设置该图标的宽高*/
	height: 15px;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>按热度排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按评价排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按兴趣排序</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="t2le">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color: transparent;
	border: none;
	text-align: center; 
	font: 25 14pt '微软雅黑 Light';
	color: rgb(255,255,255);
}
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="t2btn_search">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-search-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-search-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="t2picnoans">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>977</width>
       <height>384</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic/all.qrc">:/pic/PageB1-noans.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-close-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-close-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logoff">
   <property name="geometry">
    <rect>
     <x>1027</x>
     <y>9</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;   
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-logoff-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-logoff-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic/all.qrc"/>
 </resources>
 <connections/>
</ui>
